File: app/src/main/java/com/sistemas51/horarioslavalle/api/HourService.java
package com.sistemas51.horarioslavalle.api;

import java.util.List;

import okhttp3.ResponseBody;
import retrofit2.Call;
import retrofit2.http.GET;

public interface HourService {

    @GET("api/")
    Call<String> version();

    @GET("api/download/")
    Call<String> download();


}

File: app/src/main/java/com/sistemas51/horarioslavalle/api/HourService.kt
package com.sistemas51.horarioslavalle.api

import retrofit2.Call
import retrofit2.http.GET

interface HourService {
    @GET("api/")
    fun version(): Call<String?>?

    @GET("api/download/")
     fun download(): Call<String?>?

    @GET("api/download/")
    suspend fun downloadNew(): String
}
